uint16 nb_nodes
uint16[] trajectory_x
uint16[] trajectory_y
uint8 stop_motor
float32 arm_angle
float32 basket_angle
bool air_pump
---

